modules/libcom/src/misc/epicsTypes.h

```diff
@@ -49,15 +54,15 @@ typedef epicsUInt16     epicsEnum16;
 typedef float           epicsFloat32;
 typedef double          epicsFloat64;
 typedef epicsInt32      epicsStatus;
-
+ /** @} */
 
 typedef struct {
     unsigned    length;
     char        *pString;
 } epicsString;
 
-/*
- * !! Dont use this - it may vanish in the future !!
+/**
+ * \brief !! Dont use this - it may vanish in the future !!
```

Comment:
@anjohnson Is this warning still appropriate?  Use of `MAX_STRING_SIZE` remains common.

Reply:
I would like to see a "Don't use this" comment on the `epicsOldString` and `epicsString` typedefs, but not on the `MAX_STRING_SIZE` macro which is far too widely used to be deprecated.

Reply:
I'm happy to change this tomorrow